src/test/make/lift-directories.mk

```diff
@@ -0,0 +1,41 @@
+.EXPORT_ALL_VARIABLES:
+
+# Run from the directory basil/src/test/*/test_case/
+
+# - means continue if it doesnt exist
+-include ./config.mk
+
+NAME=$(notdir $(shell pwd))
+GIT_ROOT?=$(realpath ../../../../)
+
+GCC ?= aarch64-linux-gnu-gcc
+CLANG ?= clang
```

Comment:
In the previous script this was clang-15 so we should maintain this unless we want to update to the latest version and update all the examples to match (I don't want to do that right now unless we find something interesting with the newer version). The default package named 'clang' on ubuntu is clang 10 or something I think?